src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -99,5 +99,41 @@ public static class DefaultMessageHeaders
         ///     consumer side to determine the correct key to be used to decrypt the message.
         /// </remarks>
         public const string EncryptionKeyId = "x-encryption-key-id";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the name of the consumer who processed the
+        ///     failed message.
+        /// </summary>
+        public const string SourceConsumer = "x-source-consumer";
```

Comment:
This header contains the group id? Right? Isn't the summary misleading (and maybe the name as well)?

Reply:
Renamed and moved to `KafkaMessageHeaders`.